### ISSUE TITLE
colab cant run missing packages

### ISSUE THREAD

**Issue by xn-ml-sia** [Puzer/stylegan#38]
Hi can update the colab notebook. I think it's missing some requirements before step 1.

Thanks!

**Comment by MaskeZen** [comment]
Yes, if any can share the `requirements.txt` file I will appreciate a lot.
In my environment I cant get run it.
